website/write-translations.js

```diff
@@ -0,0 +1,31 @@
+const {i18n: {locales}} = require('./docusaurus.config.js');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+/**
+ * @param {string} locale
+ */
+async function writeTranslation(locale) {
+    const {stdout, stderr} = await exec(`npm run docusaurus -- write-translations --locale ${locale}`)
+    console.log(stdout)
+    console.error(stderr)
+}
+
+
+async function writeTranslations() {
+    for (const locale1 of locales
+        .filter(locale => locale !== 'en')) {
+        await writeTranslation(locale1);
+    }
+}
+
+module.exports = writeTranslations
+
+if (require.main === module) {
+    writeTranslations().then(() => {
+    })
```

Comment:
```suggestion
    writeTranslations()
```
ditto

Reply:
I added a `catch(e => console.error(e))` instead, it does catch meaningful error in this context because child_process.exec rejects when the underlying child process exits with non-error code.